packages/compass-e2e-tests/tests/atlas-login.test.ts

```diff
@@ -125,9 +124,7 @@ describe('Atlas Login', function () {
   });
 
   after(async function () {
-    if (TEST_COMPASS_WEB) {
-      return;
-    }
+    skipForWeb(this);
```

Comment:
What exactly is being skipped when doing this in an `after()` (or `afterEach()`) hook?

Reply:
The after() or afterEach() hook. for every before() or beforeEach() hook even if you skipped in there the corresponding after() or afterEach() hook still executes. By design because they can't know how far you got before you called this.skip().

Reply:
I just realised I am relying on skip working the same as early return there. Didn't actually check if it is even supported in after/afterEach hooks 😬

Reply:
it seems to throw and get caught. typescript somehow even knows that code after it won't execute.

Reply:
grumble: `Error: `this.skip` forbidden`. I guess I'll change those back to early return